File: algorithms/check_pallindrome_number/lua/check_palindrome.lua
local function isPalindrome(s)
    s = tostring(s)
    return s:sub(1, math.floor(#s / 2 + .5)) == s:sub(math.ceil(#s / 2 + .5)):reverse()
end
print(isPalindrome(table.concat({ ... }, "")))

File: algorithms/check_pallindrome_number/javascript/checkPalindrome.js
function isPalindrome (num) {
  let reverse = 0
  for (let i = num; i > 0; i = parseInt(i / 10)) {
    reverse = reverse * 10 + parseInt(i % 10)
  }
  if (num === reverse) console.log('Palindrome')
  else console.log('Not Palindrome')
}

isPalindrome(23432)
